app/javascript/pages/home.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Gallery, GalleryItem, Text } from '@patternfly/react-core';
+import ArticleCard from '../components/articleCard.js';
+import previewImage from '../../assets/images/img1.png';
+import colors from '../utils/colors.js';
+import { useNavigate } from 'react-router-dom';
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+const styles = {
+  headerText: {
+    fontWeight: 'bolder',
+    textAlign: 'center',
+    flex: 1,
+    color: colors.appDarkBlue,
+    fontSize: '50px'
+  }
+};
+const Home = () => {
+  const navigate = useNavigate();
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    axios
+      .get('/articles', {
+        headers: {
+          'content-type': 'application/json'
```

Comment:
I guess this one is not needed after you used `axios.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
`

Reply:
Yes, thank you, updated it